airbyte-ci/connectors/pipelines/pipelines/gradle.py

```diff
@@ -64,8 +65,8 @@ def _get_gradle_command(self, extra_options: Tuple[str] = ("--no-daemon", "--sca
         command = (
             ["./gradlew"]
             + list(extra_options)
-            + [f":airbyte-integrations:connectors:{self.context.connector.technical_name}:{self.gradle_task_name}"]
-        )
+            + [f":airbyte-integrations:connectors:{self.context.connector.technical_name}:{self.gradle_task_name}"] if hasattr(self.context, "connector") else ["./gradlew", ":airbyte-cdk:java:airbyte-cdk:build"]
+        ) # NOTE: obviously not the ideal way to do this, just wanted to get something that ran
```

Comment:
🙏 please override `_get_gradle_command` in your subclass before merging.

Reply:
For sure!